lib/utils/config.js

```diff
@@ -7,6 +7,7 @@ function config(options){
     return require(configFile);
   }
   else {
+    Ember.deprecate('Relying on default configuration is deprecated', true);
```

Comment:
We can't use `Ember.deprecate` here, because it is ran in node-land (and `Ember` global is not available).  We should use something like:

```javascript
console.warn('"config/ember-try.js" is missing from your project, please run "ember generate ember-try".');
```